### PR TITLE
chore: release 0.32.0

### PR DESCRIPTION
**Pull request by guilhem-barthes**
Signed-off-by: Guilhem Barthes <guilhem.barthes@owkin.com>

- [x] [changelog](../CHANGELOG.md) was updated with notable changes
- [x] documentation was updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]()
 
+## [0.32.0](https://github.com/Substra/orchestrator/releases/tag/0.32.0) - 2023-01-09
```
Comment (ThibaultFy) [inline CHANGELOG.md:9]: Normally we update the 0.32.1 to keep the same number of main version between main release

Comment (ThibaultFy) [reply CHANGELOG.md:9]: 0.31.1 in that case

Comment (guilhem-barthes) [reply CHANGELOG.md:9]: I am not sure to understand. The previous one was 0.31.0, why should we increase both major and bugfix number?

Comment (ThibaultFy) [reply CHANGELOG.md:9]: Yes my bad, should be 0.31.1

**Comment by guilhem-barthes** [comment]
Wrong version (0.32.0 instead of 0.31.1)

**Pull request closed without merging**